app/renderer/pages/index.tsx

```diff
@@ -1,9 +1,119 @@
-import { Spinner, Stack } from '@chakra-ui/react';
+import { Alert, AlertIcon, Avatar, Box, Button, FormControl, FormErrorMessage, FormLabel, Input, Modal, ModalBody, ModalCloseButton, ModalContent, ModalHeader, ModalOverlay, Stack, VStack } from '@chakra-ui/react';
+import { useMutation } from '@tanstack/react-query';
+import { Formik, Field } from 'formik';
+import { useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+import fetcher from '../src/axios';
+import { useUser } from '../src/providers/user-provider';
+import { UserUpdatedResponse } from '../src/types/user';
+import { OnboardingSchema } from '../src/validation/onboarding-schema';
 
 const Home = () => {
+  const [uploadedPicture, setUploadedPicture] = useState<File>(null);
+  const { user, showOnboardingModal, setUser } = useUser();
+  const { mutateAsync, isLoading, error, isError } = useMutation(
+    ['/update-user'],
+    async (formData : FormData) => fetcher.put(`/users/${user.id}`, formData, { headers: { 'Content-Type': 'multipart/form-data' } }),
+    { onSuccess: (data: UserUpdatedResponse) => {
+      const { data: { firstName, lastName, profilePicture } } = data;
+      if (data.data.profilePicture){
+        setUser({ ...user, firstName, lastName, profilePicture });
+      } else {
+        setUser({ ...user, firstName, lastName });
+      }
+    } },
+  );
+  const onDrop = useCallback((acceptedFile) => {
+    setUploadedPicture(acceptedFile[0]);
+  }, []);
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop,
+    maxFiles: 1,
+    accept: {
+      'image/jpeg': [],
+      'image/png': [],
+    } });
```

Comment:
Reformat this code a little so it's more readable.

```suggestion
  const [uploadedPicture, setUploadedPicture] = useState<File>(null);
  const { user, showOnboardingModal, setUser } = useUser();
    const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop,
    maxFiles: 1,
    accept: {
      'image/jpeg': [],
      'image/png': [],
    } });
    
    const onDrop = useCallback((acceptedFile) => {
    setUploadedPicture(acceptedFile[0]);
  }, []);
    
  const { mutateAsync, isLoading, error, isError } = useMutation(
    ['/update-user'],
    async (formData : FormData) => fetcher.put(`/users/${user.id}`, formData, { headers: { 'Content-Type': 'multipart/form-data' } }),
    { onSuccess: (data: UserUpdatedResponse) => {
      const { data: { firstName, lastName, profilePicture } } = data;
      if (data.data.profilePicture){
        setUser({ ...user, firstName, lastName, profilePicture });
      } else {
        setUser({ ...user, firstName, lastName });
      }
    } },
  );
```